vtex-sites/nextjs.store
# refactor: Extract `CartItem` from the starter to `@faststore/ui`

## What's the purpose of this pull request?

Extract the `CartItem` component from our starter to `@faststore/ui`.

## How to test it?

Visit https://sfj-4a90f81--nextjs.preview.vtex.app/4k-philips-monitor-99988213/p and add a product to the cart. A `CartItem` component will be rendered inside the mini-cart sidebar. It should look and work the same as before.

<img src="https://user-images.githubusercontent.com/381395/189658104-c64012ee-dd16-4ce0-a2bc-4535e8bec5d5.gif" width="50%" alt="Before and after changes" />

## References

- https://github.com/vtex/faststore/pull/1461

## Checklist

- [x] Once the new `@faststore/ui` version is released, commit the `bun.lockb`.
